bedrock/settings/base.py

```diff
@@ -584,6 +597,25 @@ def language_url_map_with_fallbacks():
 if DEBUG:
     STATICFILES_DIRS += (path("media"),)
 
+GS_OBJECT_PARAMETERS = {
+    "cache_control": "max-age=2592000, public, immutable"  # 2592000 == 30 days / 1 month
+}
+
+if GS_BUCKET_NAME and GS_PROJECT_ID:
+    GS_CUSTOM_ENDPOINT = MEDIA_URL  # hostname that proxies the storage bucket
+    GS_FILE_OVERWRITE = False
+    GS_LOCATION = "custom-media"  # path within the bucket to upload to (and used when generating URLs)
```

Comment:
`GS_LOCATION` and `GS_CUSTOM_ENDPOINT` are additions here - the rest is refactoring. 

you can see docs for these settings at https://django-storages.readthedocs.io/en/latest/backends/gcloud.html